.github/workflows/wheels.yml

```diff
@@ -0,0 +1,129 @@
+# Workflow to build and test wheels
+name: Wheel builder
+
+on:
+  schedule:
+    # Nightly build at 3:42 A.M.
+    - cron: "42 3 */1 * *"
+  push:
+    tags:
+      # Final releases, release candidates and post-releases build
+      - "[0-9]+.[0-9]+.[0-9]+(rc[0-9]+)?(.post[0-9]+)?"
+  pull_request:
+    branches:
+      - master
+      - "[0-9]+.[0-9]+.X"
+
+jobs:
+  build_wheels:
```

Comment:
I think this works to trigger based on commit:

```suggestion
  build_wheels:
    if: "contains(github.event.head_commit.message, '[build wheels]')"
```

Reply:
I have previously tried it, but it seems that the `github.event.head_commit.message` no longer exists for PRs. See: https://developer.github.com/webhooks/event-payloads/#pull_request.

Reply:
I think that action is used to lint commit messages.

For now, we can get this working with cron first and then work on getting triggering with a commit message to work later.